### PR TITLE
CSI: volume init command

### PR DESCRIPTION
**Pull request by tgross**
Create a convenience command for generating example CSI volume specifications,
similar to the existing `nomad job init` or `nomad quota init` commands.

### CODE REVIEW

File: command/volume_init.go
Comment (notnoop) [inline command/volume_init.go:107]: Have you considered making these an asset?  We migrated to using assets in `command/assets/` for managing example jobs?

File: command/volume_init.go
Comment (notnoop) [inline command/volume_init.go:75]: Should it take an optional argument to specify the path? `job init` started accepting the parameter.

**Review by notnoop — APPROVED** [review]
Few comments about having parity with `job init`. I'd defer to your judgement, and feel free to merge as you see fit without a second review.

**Comment by tgross** [comment]
> I see that `quota init` didn't pick up the changes that `job init` had. Might make sense to update that one too.

Yeah, and this code was pretty much a direct copy-and-paste of the `quota init` code, which is why it doesn't have those features either. My inclination is to merge it as-is and then I'll open an issue to fill out both `quota init` and `volume init`; that way we can use this to help out E2E testing now and after the private beta window I can come back to do some fit-and-finish work.

**Comment by tgross** [comment]
Opened https://github.com/hashicorp/nomad/issues/10198

**Pull request merged**
